### DISCUSSION TITLE
LSTC dummies

### DISCUSSION THREAD

**Discussion by markrhacker** [OpenRadioss/OpenRadioss#2242] [Q&A]
Hello
Has anybody successfully got either the rigid or fast LSTC dummies to convert and run?
Thanks !

**Comment by hdrz** [comment]
Even if the conversion is successful, the dummy has to be re-certified for use with radioss, since the elements and keywords are different and the solver is different. 
It can be done of-course, look at the pdf file that comes with the dummy, the certification procedure is outlined there. 

Note: The certification is needed in order to believe the results you get from the dummy.

**Reply by markrhacker** [reply]
Understand - but I can't even get it to run

**Reply by hdrz** [reply]
Yeah I would expect so, since the models have many parts which include only inertia and no elements, not sure how they translate to radioss. Also accelerometers and such, probably not everything supported yet by the radioss reader.

**Reply by markrhacker** [reply]
thats make a lot sense - thanks

**Reply by markrhacker** [reply]
Is there a ATD that does work with OpenRadioss ?

**Reply by hdrz** [reply]
I think so, the radioss Humanetics dummies should work, but they cost $$$$$!

**Reply by markrhacker** [reply]
Nothing simpler and OpenSource?

**Reply by elequiniou** [reply]
Hi, 

Thanks a lot for your interest in OpenRadioss project!
You are right, high-end Humanetics dummies are available for Radioss commercial version.
For open-source solution, I advise you to attend this online event, OVTO User Meeting, to learn about running open-source Viva+ models with OpenRadioss, registration link [https://www.linkedin.com/events/viva-usermeeting7143601294430146560/comments/](https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.linkedin.com%2Fevents%2Fviva-usermeeting7143601294430146560%2Fcomments%2F&data=05%7C02%7Celequiniou%40altair.com%7C0628633a2b7a4ffea0b308dc105addcb%7C2bae5b570eb848fbba47990259da89d2%7C0%7C0%7C638403229697657707%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=HZCLaknVTPX8L8Q0w74l0%2Boxw4TE%2F7%2BVccH3mXh0hp0%3D&reserved=0)
We are also collaborating with partners on THUMS and Piper open-source human body models.
For ATD, I advise you to have a look at https://github.com/OpenRadioss/ModelExchange/tree/main/Safety, hoping we will be able to rapidly increase this collaborative data base of free models with the help of the community.

Kind Regards,

Eric.

**Reply by markrhacker** [reply]
Thanks Eric - I've tried a few of those, none are OOT.  I've joined the event already.  
I used in the model in the ModelExchange - as a simple starter
Thanks
